awx/settings/defaults.py

```diff
@@ -109,6 +109,8 @@ def IS_TESTING(argv=None):
 # Examples: "http://media.lawrence.com", "http://example.com/media/"
 MEDIA_URL = '/media/'
 
+LOGIN_URL = '/api/login/'
+
```

Comment:
Also need `LOGIN_REDIRECT_URL` and `LOGOUT_REDIRECT_URL`, right?

Reply:
Probably not, I think that's left to UI.